### ISSUE TITLE
Links generated do not pull podcasts 

### ISSUE THREAD

**Issue by mja292** [fpnewton/DailyWirePodcastProxy#36]
Ive followed all steps and generated the links for the shows, but when I paste the links into pocket casts they do not find the shows. Help please I just spent two hours on this.

**Comment by martinlumi** [comment]
You have to submit the links here, and get a new Pocket Cast link that can be added in the app.

https://pocketcasts.com/submit/

**Comment by mja292** [comment]
> You have to submit the links here, and get a new Pocket Cast link that can be added in the app.
> 
> 
> 
> https://pocketcasts.com/submit/

Yes that is where I run into the issue. When submitting a link to this page it fails.

**Comment by martinlumi** [comment]
I see. What does your URLs look like. If you run the program from your PC, it will host it locally and that will not work with Pocket Casts. I run the program on Google Cloud, which gives me a public IP/URL that I use for Pocket Casts.

**Comment by mja292** [comment]
Thank you for the info!!!! 

The links look fine, exactly what I would expect. But they fail on pocketcasts. I am running in docker locally - it seems like the only way I could make it work. Could you share details other how you run it on the cloud? Im semi new to all this.

**Comment by martinlumi** [comment]
If your URLs look something like "192.168.X.XXX" or "10.0.1.X", this means you are running the software locally on your computer. This will work fine for adding it to Apple Podcasts and certain other podcast apps (as long as you're on the same wifi as the computer).

However, when you submit a link to Pocket Casts, it has to be readable publicly. In my case, I opted for using Google Cloud Virtual Machine to run a Ubuntu server that in turn runs the proxy program. This gives me a public IP that I can paste into Pocket Casts' submit function, and get a Pocket Casts URL.

This does require some setup, and I also have a UP-monitor that notifies me if the server for some reason goes down. You can find guides for all of this. I used ChatGPT as a sparring partner to get this done.
